docker-compose.yaml

```diff
@@ -8,38 +8,65 @@ services:
     expose:
       - 5000
     tty: true
+    restart: always
+    depends_on:
+      - db
     networks:
       app-network:
         ipv4_address: 172.22.0.6
-
   nginx:
     image: nginx:latest
-    container_name: nginx
+    container_name: nginx-service
     build:
       context: .
       dockerfile: Dockerfile.nginx
     ports:
       - "80:80"
+      - "443:443"
+    restart: always
+    tty: true
     depends_on:
       - authz
+      - certbot
     networks:
       app-network:
         ipv4_address: 172.22.0.5
     volumes:
-      - nginx_log:/var/log/nginx
-
+      - /var/authz/log/nginx_log:/var/log/nginx
+      - /var/authz/letsencrypt:/etc/letsencrypt/
+  certbot:
+    container_name: certbot-service
+    build:
+      context: .
+      dockerfile: Dockerfile.certbot
+    tty: true
+    restart: always
+    expose:
+      - 80
+    volumes:
+      - /var/authz/letsencrypt:/etc/letsencrypt
+      - /var/authz/log/certbot_log:/var/log/letsencrypt
+    networks:
+      app-network:
+        ipv4_address: 172.22.0.7
+  db:
+    image: postgres:latest
+    container_name: Postgres
+    restart: always
+    env_file:
+      - /tmp/db_login.env
```

Comment:
how should i run this `docker-compose` locally

Reply:
+ 
```
TASK [service_authz : Build container (docker-compose)] *******************************************************************************************
fatal: [3.123.23.80]: FAILED! => {"changed": false, "msg": "Configuration error - Couldn't find env file: /tmp/db_login.env"}
```

Reply:
Fixed. 
If you want to run this docker-compose locally, you need to run decrypt_secrets.sh before launching docker-compose
🤞